src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -282,30 +282,50 @@ export class CalcitePickListItem {
     ) : null;
   }
 
-  render(): VNode {
+  renderLabelContent(): VNode {
     const { description, label } = this;
 
+    return (
+      <div
+        aria-checked={this.selected.toString()}
+        class={CSS.textContainer}
+        role="menuitemcheckbox"
+      >
+        <span class={CSS.title}>{label}</span>
+        {description ? <span class={CSS.description}>{description}</span> : null}
+      </div>
+    );
+  }
+
+  render(): VNode {
+    const { label } = this;
+
     return (
       <Fragment>
         {this.renderIcon()}
         {this.renderActionsStart()}
-        <label
-          aria-label={label}
-          class={CSS.label}
-          onClick={this.pickListClickHandler}
-          onKeyDown={this.pickListKeyDownHandler}
-          ref={(focusEl): HTMLLabelElement => (this.focusEl = focusEl)}
-          tabIndex={0}
-        >
-          <div
-            aria-checked={this.selected.toString()}
-            class={CSS.textContainer}
-            role="menuitemcheckbox"
+        {this.nonInteractive ? (
+          <label
```

Comment:
If this item is non-interactive, I don't think it needs the `aria-label`, `onclick`, `onkeydown`, or `<label>` element. Is that right? It may not need to be focused either. I'm not sure on that one.

The content wouldn't need a checkbox role or aria-checked, right? I think the role would just be `menuitem` and not `menuitemcheckbox` or maybe not even be a menuitem if it doesn't need to be focused at all.

Reply:
I think you're right on that @driskull. But we can discuss in a quick chat...?